compiler.pri

```diff
@@ -226,6 +226,29 @@ unix {
 		QMAKE_OBJECTIVE_CFLAGS *= -O3 -march=native -ffast-math -ftree-vectorize -fprofile-use
 		QMAKE_OBJECTIVE_CXXFLAGS *= -O3 -march=native -ffast-math -ftree-vectorize -fprofile-use
 	}
+
+	CONFIG(c++11) {
+		# Qt 5 will pass the expected compiler flags
+		# needed for C++11 mode when CONFIG includes c++11.
+		# But Qt 4 won't, so we add it manually.
+		lessThan(QT_MAJOR_VERSION, 5) {
+			QMAKE_CXXFLAGS *= -std=c++11
+		}
+
+		# ZeroC Ice's C++11 libs are in /usr/lib/$triple/c++11 on Debian.
+		# This convention may apply to other software as well.
```

Comment:
Not sure what this is trying to say. What does it matter here if other software does that?
If we depend on it elsewhere we will have to make additional adjustments for that.
If other software does, we don't care.